imports/server/accounts.ts

```diff
@@ -58,6 +59,11 @@ Accounts.registerLoginHandler((options: LoginOptions) => {
     return undefined;
   }
 
+  check(options, {
```

Comment:
If I'm reading this correctly, won't the `options` object always have an `isGoogleJrLogin` field, and therefore have a key which is not specified in this object, and therefore always fail this `check()`?

Reply:
Hmm. Yeah, I think that's right. Fixed